matRadGUI.m

```diff
@@ -1748,7 +1748,6 @@ function btnuiTableAdd_Callback(hObject, ~, handles)
 data{sEnd+1,2} = 'Select VOI Type';
 data{sEnd+1,3} = 2;
 data{sEnd+1,4} = 'Select obj func/constraint';
-data{sEnd+1,6} = '';
```

Comment:
Indeed, this was my first solution. But if you want to initialize consistently you have to initialize all columns, right? I would prefer to set here only non-empty columns